### PR TITLE
Add deepgraph package for Python

### PR DESCRIPTION
**Pull request by dominiktraxl**

### CODE REVIEW

**Comment by dominiktraxl** [comment]
The package includes two cythonized .c extensions. The .c files are contained in the distributed tar.gz. file on pypi.

File: recipes/deepgraph/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+  number: 0
+  skip: True  # [py33]
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - numpy
+    - pandas
```
Comment (synapticarbors) [inline recipes/deepgraph/meta.yaml]: It doesn't look like you need `pandas` to actually build the package.

File: recipes/deepgraph/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+build:
+  number: 0
+  skip: True  # [py33]
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - numpy
```
Comment (synapticarbors) [inline recipes/deepgraph/meta.yaml]: Change to `numpy x.x` both in the `build` and `run` sections. See:

https://github.com/conda-forge/staged-recipes#4-what-does-numpy-xx-mean

**Comment by synapticarbors** [comment]
@dominiktraxl -- Thanks for your contribution to conda-forge. A feedstock for this recipe should be generated soon and you'll be receiving several automated messages about it. When the feedstock has been rendered, please take a look at the README in that repo for information about updating and maintaining the recipe. One of the key points is all updates should be done by forking that repo and following a standard pull request workflow. If you haven't looked at it already the docs are quite helpful:

https://conda-forge.org/docs/

If you run into any difficulties please let us know.

**Pull request merged**